utils/notifications_utils/template.py

```diff
@@ -605,6 +606,8 @@ def do_nice_typography(value):
         remove_whitespace_before_punctuation
     ).then(
         make_quotes_smart
+    ).then(
+        remove_smart_quotes_from_email_addresses
```

Comment:
Rename `remove` to `replace`? I thought it was actually just removing until I saw the logic.

Reply:
@andrewoh531 you're probably right but I'm hesitant to do it simply to avoid diverging where possible.

Reply:
@andrewoh531 at some stage we should make a call for how we want to approach upstream and diverging, etc. but not yet.